package.json

```diff
@@ -72,22 +72,22 @@
 		"@types/jsdom": "^16.2.13",
 		"@types/node": "^13.1.8",
 		"@types/parse5": "^6.0.2",
-		"@typescript-eslint/eslint-plugin": "^6.19.0",
-		"@typescript-eslint/parser": "^6.19.0",
+		"@typescript-eslint/eslint-plugin": "^8.6.0",
+		"@typescript-eslint/parser": "^8.6.0",
 		"eslint": "^8.56.0",
 		"eslint-config-prettier": "^9.1.0",
 		"eslint-plugin-prettier": "^5.1.3",
 		"mocha": "^5.2.0",
 		"prettier": "^3.2.3",
 		"safe-publish-latest": "^1.1.4",
-		"typescript": "^5.1.6"
+		"typescript": "^5.5.4"
 	},
 	"prettier": {
 		"singleQuote": true,
 		"arrowParens": "avoid",
 		"printWidth": 100
 	},
 	"engines": {
-		"node": ">= 12 || ^11.10.1 || ^10.13 || ^8.10"
+		"node": ">= 18"
```

Comment:
even if nothing else changed, this alone requires a semver-major bump, fwiw

Reply:
Yeah, that's why I said "This is a breaking change".

Reply:
gotcha, the "so whatever" implied you weren't going to publish it as one.

Reply:
Aha, I see how that was misleading; I just meant that it's unlikely to break anyone in practice.